src/queryAnalysis/config/queryConfigs.go

```diff
@@ -5,114 +5,125 @@ import "github.com/newrelic/nri-mssql/src/queryAnalysis/models"
 	{
 		Name: "MSSQLTopSlowQueries",
-		Query: `DECLARE @IntervalSeconds INT = %d; 		-- Define the interval in seconds
-				DECLARE @TopN INT = %d; 				-- Number of top queries to retrieve
-				DECLARE @ElapsedTimeThreshold INT = %d; -- Elapsed time threshold in milliseconds
-				DECLARE @TextTruncateLimit INT = %d; 	-- Truncate limit for query_text
-				
-				WITH RecentQueryIds AS (
-					SELECT  
-						qs.query_hash as query_id
-					FROM 
-						sys.dm_exec_query_stats qs
-					WHERE 
-						qs.execution_count > 0
-						AND qs.last_execution_time >= DATEADD(SECOND, -@IntervalSeconds, GETUTCDATE())
-						AND qs.sql_handle IS NOT NULL
-				),
-				QueryStats AS (
-					SELECT
-						qs.plan_handle,
-						qs.sql_handle,
-						LEFT(SUBSTRING(
-							qt.text,
-							(qs.statement_start_offset / 2) + 1,
-							(
-								CASE
-									qs.statement_end_offset
-									WHEN -1 THEN DATALENGTH(qt.text)
-									ELSE qs.statement_end_offset
-								END - qs.statement_start_offset
-							) / 2 + 1
-						), @TextTruncateLimit) AS query_text, 
-						qs.query_hash AS query_id,
-						qs.last_execution_time,
-						qs.execution_count,
-						(qs.total_worker_time / qs.execution_count) / 1000.0 AS avg_cpu_time_ms,
-						(qs.total_elapsed_time / qs.execution_count) / 1000.0 AS avg_elapsed_time_ms,
-						(qs.total_logical_reads / qs.execution_count) AS avg_disk_reads,
-						(qs.total_logical_writes / qs.execution_count) AS avg_disk_writes,
-						CASE
-							WHEN UPPER(
-								LTRIM(
-									SUBSTRING(qt.text, (qs.statement_start_offset / 2) + 1, 6)
-								)
-							) LIKE 'SELECT' THEN 'SELECT'
-							WHEN UPPER(
-								LTRIM(
-									SUBSTRING(qt.text, (qs.statement_start_offset / 2) + 1, 6)
-								)
-							) LIKE 'INSERT' THEN 'INSERT'
-							WHEN UPPER(
-								LTRIM(
-									SUBSTRING(qt.text, (qs.statement_start_offset / 2) + 1, 6)
-								)
-							) LIKE 'UPDATE' THEN 'UPDATE'
-							WHEN UPPER(
-								LTRIM(
-									SUBSTRING(qt.text, (qs.statement_start_offset / 2) + 1, 6)
-								)
-							) LIKE 'DELETE' THEN 'DELETE'
-							ELSE 'OTHER'
-						END AS statement_type,
-						CONVERT(INT, pa.value) AS database_id,
-						qt.objectid
-					FROM
-						sys.dm_exec_query_stats qs
-						CROSS APPLY sys.dm_exec_sql_text(qs.sql_handle) AS qt
-						JOIN sys.dm_exec_cached_plans cp ON qs.plan_handle = cp.plan_handle
-						CROSS APPLY sys.dm_exec_plan_attributes(cp.plan_handle) AS pa
-					WHERE
-						qs.query_hash IN (SELECT DISTINCT(query_id) FROM RecentQueryIds)
-						AND qs.execution_count > 0
-						AND pa.attribute = 'dbid'
-						AND DB_NAME(CONVERT(INT, pa.value)) NOT IN ('master', 'model', 'msdb', 'tempdb')
-						AND qt.text NOT LIKE '%%sys%%'
-						AND qt.text NOT LIKE '%%INFORMATION_SCHEMA%%'
-						AND qt.text NOT LIKE '%%schema_name()%%'
-						AND qt.text IS NOT NULL
-						AND LTRIM(RTRIM(qt.text)) <> ''
-				)
+		Query: `DECLARE @IntervalSeconds INT = %d;
```

Comment:
PR should ideally be showing only the changed part and not the whole query. Let's make sure even the indentation is not disturbed while creating PRs so that we can review mainly what is changed exactly. @avadla